src/index.ts

```diff
@@ -1,9 +1,21 @@
 declare module '@webcomponents/custom-elements';
 
-(async function () {
-  if (!('customElements' in self)) {
-    await import('@webcomponents/custom-elements');
-  }
-
+if (typeof PRERENDER === 'boolean') {
```

Comment:
Add a comment here to explain what's happening?

Is it correct that this block should happen if `PRERENDER` is `false`?

Reply:
fwiw `'PRERENDER' in self` would be more consistent with the `else if`.

Reply:
TBH this is a workaround. Critters is supposed to set `PRERENDER` to `true` during prerendering and `false` for the runtime build, but the latter isn't working.  I'll try to get that fixed upstream and correct it here. It should just be as simple as:

```
if (PRERENDER) {

}
```

Webpack's definePlugin doesn't support static property existence checks, so `in` doesn't work here.

Reply:
Re-affirmed that it is in fact not possible to modify parser plugins once the build has begun, making it impossible for prerender-loader to set `PRERENDER=true` / `PRERENDER=false`.  I've stripped out all the conditional logic instead, since it belonged in prerender-loader itself anyway. We're clean and there's no more conditional stuff!